server/models/delius/deliusServiceUser.ts

```diff
@@ -11,11 +11,28 @@ interface OffenderLanguages {
   primaryLanguage: string
 }
 
+interface Address {
```

Comment:
when deserializing responses, does it matter if the field is missing? will it just cast it to null?

Reply:
Yeah, if the field might not be present at all in the response then these should be represented as `addressNumber?` etc, I think.

Reply:
Have updated this - I think we used ` | null` for most of the values in the past, but it probably should be `undefined` instead if it's not in the response at all (which is the case).